packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -31,6 +36,43 @@ import { ScreenVariation } from './screen-variations';
 import ScreenBorder from './screen-border';
 import StyleBook from '../style-book';
 import ScreenCSS from './screen-css';
+import { unlock } from '../../experiments';
+
+const SLOT_FILL_NAME = 'GlobalStylesMenu';
+const { Slot: GlobalStylesMenuSlot, Fill: GlobalStylesMenuFill } =
+	createSlotFill( SLOT_FILL_NAME );
+
+function GlobalStylesMenu() {
```

Comment:
Nitpick - I personally think of global styles as one big menu system, so the component could be clearer about what it represents. Maybe something like `GlobalStylesActionMenu`.

Reply:
Done.